### DISCUSSION TITLE
<Is Github aware of the recent spammers & scammers on the platform?>

### DISCUSSION THREAD

**Discussion by Hakorehdeh** [community/community#70013] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Lately, I see a lot of spammers taking advantage of the Github community to promote their scams and deceptions. At first I decided I would take it upon myself to report these people anytime I see them. But lately, I realized the scammers are not even leveraging on repositories to promote their scams but on topics. They simply create a new account under the topic and they're back. According to my research, a bunch of Github topics are currently ranking on Google search and if any repository appears under those topics, they benefit from the rank. 

Below are a few of the topics used by the scammers:

1. https://github.com/topics/free-eshop-codes-nintendo-switch-redeem-code-lists 
2. https://github.com/topics/monopoly-go-unlimited-dice-android
3. https://github.com/topics/onlyfans-viewer
4. https://github.com/topics/hack-instagram-account-2023-insta-hacking-tool-new

If you click any of these topics, you'd definitely find some spam repositories from spam accounts under them. It appears, we ordinary users can only do our part by reporting them but it'd never be enough. I've had a couple of instances where the spammers returned withing 5 minutes after their accounts was taken down. I hope the admin finds this and do something about it.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Hakorehdeh 👋🏼  We really appreciate you flagging this and reporting bad actors. For anyone else looking for spam resources, the best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!

**Reply by davidar** [reply]
My https://github.com/explore page is currently full of these spam topics, so obviously it's still an ongoing issue. As @Hakorehdeh notes, reporting individual users who are promoting these topics isn't going to do much, and there doesn't seem to be any way to report entire topics either.

**Reply by Hakorehdeh** [reply]
And it persists:
https://github.com/topics/free-google-play-gift-card-codes-generator-redeem
https://github.com/topics/free-eshop-codes-nintendo-switch-redeem-code-lists
https://github.com/topics/view-private-instagram-without-human-verification

I've reported a couple of them already, but I'm pretty sure they'd be back in no time.
